### DISCUSSION TITLE
Is E2E implemented?

### DISCUSSION THREAD

**Discussion by darkjake** [nkonev/videochat#5] [General]
Encryption is important to me, Is E2E implemented?

**Comment by nkonev** [comment]
By e2e I meant end-to-end tests.
There is no message's content encryption.

**Reply by darkjake** [reply]
aw nuts :(
do you take suggestions? If so auto PGP would be killer, its the one thing keeping me on mattermost.

**Reply by nkonev** [reply]
Thanks for idea, but I cannot guarantee now that I'm going to implement things like this. It is a project which I write in spare time, still without global plans.

By the way, it's open source under a permissive license, so you can fork it and make what you want.
